client/src/components/NavLinkStyled.js

```diff
@@ -1,11 +1,7 @@
 import styled from '@emotion/styled';
-import React from 'react';
+import { NavLink } from 'react-router-dom';
```

Comment:
I'm not sure if you could import `NavLink` from react-router-dom
Maybe you mean `Link`?

Reply:
Thank you. But you can import this component. Checkout: https://reacttraining.com/react-router/web/api/NavLink